content/en/docs/concepts/scheduling-eviction/kube-scheduler.md

```diff
@@ -95,3 +95,6 @@ of the scheduler:
 * Learn about [configuring multiple schedulers](/docs/tasks/administer-cluster/configure-multiple-schedulers/)
 * Learn about [topology management policies](/docs/tasks/administer-cluster/topology-manager/)
 * Learn about [Pod Overhead](/docs/concepts/configuration/pod-overhead/)
+* Learn about scheduling of Pods that use volumes in:
```

Comment:
Should volume topology be here too? I guess though, there's a lot of scheduler plugins beyond storage. Do we want to list them all out here? cc @ahg-g

Reply:
Where is volume topology documented, or more specifically, what should I link to here?

Reply:
https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode and allowedtopologies below that are probably the best sections to link

Reply:
Okay, added.

Reply:
It does sound like it might be worth filing an issue about volume topology. @pohly can you describe the page you were hoping to find, in an issue against [k/website](https://github.com/kubernetes/website)?

Reply:
Done, see https://github.com/kubernetes/website/issues/22506